packages/opc-styles/package.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "@one-platform/opc-styles",
+  "version": "0.0.1",
+  "description": "This packages contains the all common styles which are important to get #### started with any component which includes global variables, colors, common mixin",
+  "main": "opc-styles.scss",
+  "scripts": {
+    "test": "./node_modules/.bin/node-sass -w ./test/ -o css/"
```

No need to provide the full path here, as npm scripts automatically loads the `.bin` into $PATH.

```suggestion
    "test": "node-sass -w ./test/ -o css/"
```